src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int NUMBER = 10;
+    static final String WHITE = "WHITE";
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figure = random.nextInt(5);

```

Comment:
нам також треба винести цифру 5 в константу
```suggestion
        int randomFigureNumber = random.nextInt(NIMBER_OF_FIGURES);

```